### DISCUSSION TITLE
A way of cusomizing new created profiles

### DISCUSSION THREAD

**Discussion by Merlinux-source** [filips123/PWAsForFirefox#203] [Ideas]
First of all, I would love to see an option which enables auto creation of an profile per web app. But that's not the thing I want to talk about. I want to talk about customized profiles. 
Everytime I install a webapp in a new profile, firefox asks me if I want to share infos with firefox. and I have to install the same addons which I need on almost every website again. How about making pre configured new profiles an option? I imagine this as a folder created by the PWA backend, which contains a default profile, which I can load into firefox, customize it and then use it as a new default profile.
Is this hard to implement? and would one of the developers (I don't know how many there are) this as a valuable feature to implement?

**Comment by filips123** [comment]
I agree that would be nice and probably shouldn't be too hard to implement. There could just be a folder that is copied into a newly created profile. The only thing I'm not sure about is if Firefox profiles contain any settings that use hard-coded paths and would break when changing the profile to another folder.
